### PR TITLE
Send keys test

### PR DESCRIPTION
**Pull request by jangromko**
Added test for sending text to active element with `send_keys/2`.

Enabled only for Chromedriver, because new versions of Selenium don't support this action. Test should be also useful for implementing W3C webdriver protocol.

### CODE REVIEW

File: integration_test/cases/browser/send_text_to_active_element_test.exs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+defmodule Wallaby.Integration.Browser.SendTextToActiveElementTest do
```
Comment (michallepicki) [inline integration_test/cases/browser/send_text_to_active_element_test.exs]: Can we rename it to `send_keys` from `send_text`? This one and the other existing test in browser, not sure why it was `send_text`

Comment (jangromko) [reply integration_test/cases/browser/send_text_to_active_element_test.exs]: OK, sure!

Comment (jangromko) [reply integration_test/cases/browser/send_text_to_active_element_test.exs]: Done. :)

Comment (michallepicki) [reply integration_test/cases/browser/send_text_to_active_element_test.exs]: Awesome, can you also rename `integration_test/cases/browser/send_text_test.exs` to `integration_test/cases/browser/send_keys_test.exs` while at it?

Comment (jangromko) [reply integration_test/cases/browser/send_text_to_active_element_test.exs]: Also done, @michallepicki

**Pull request merged**